<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1454</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Монитор</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>620</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="History">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>851</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="GraphResult" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1431</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="ConnectorTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Устройство</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PID</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>680</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск устройств</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stop">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>650</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Завершить</string>
    </property>
   </widget>
   <widget class="QLabel" name="posX">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>600</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>posX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataX">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>620</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="posY">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>640</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>posY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataY">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>660</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1454</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="separator"/>
    <addaction name="SaveAction"/>
    <addaction name="SaveAsAction"/>
    <addaction name="separator"/>
    <addaction name="ExitAction"/>
   </widget>
   <widget class="QMenu" name="Settings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="ControllerSettingsAction"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="AboutAction"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Settings"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="SaveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="SaveAsAction">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="ExitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="ControllerSettingsAction">
   <property name="text">
    <string>Настройка микроконтроллера</string>
   </property>
   <property name="toolTip">
    <string>Настройка микроконтроллера</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="text">
    <string>Открыть </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
